Pursuit-Core-iOS-Episodes-from-Online/Pursuit-Core-iOS-Episodes-from-Online/Controllers/EpisodesViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  EpisodesViewController.swift
+//  Pursuit-Core-iOS-Episodes-from-Online
+//
+//  Created by Malcolm S. Turnquest on 9/12/19.
+//  Copyright © 2019 Benjamin Stone. All rights reserved.
+//
+
+import UIKit
+
+class EpisodesViewController: UIViewController {
+
+     var show: TVShowsWrapper?
+        @IBOutlet weak var showImage: UIImageView!
+        @IBOutlet weak var showName: UILabel!
+        @IBOutlet weak var showDescrip: UITextView!
+        
+        var episodes = [Episodes](){
+            didSet {
+                episodeTable.reloadData()
+              setUpLabelsShow()
+            }
+        }
+        @IBOutlet weak var episodeTable: UITableView!
+        override func viewDidLoad() {
+            super.viewDidLoad()
+            episodeTable.delegate = self
+            episodeTable.dataSource = self
+            getData()
+        }
+        func getData(){
+            Episodes.getEpisode(id: (show?.show.id)!){ (result) in
+        switch result {
+        case .failure(let error):
+        print(error)
+        case .success(let episode):
+        DispatchQueue.main.async{
+        return self.episodes = episode
+        }
```

Comment:
Good job with the code on this VC, everything seems to work well. Just make sure to indent these brackets so the function is more organized.